back/src/api/common/services/tobacco/tobaccoTaxCalculator.ts

```
@@ -0,0 +1,144 @@
+  }
+
+  private static getReadableTypeName(type: string): string {
+    const names: Record<string, string> = {
+      tobaccoCigarettes: 'Cigarettes',
+      tobaccoCigarillos: 'Cigarillos',
+      tobaccoCigars: 'Cigares',
+      tobaccoOther: 'Tabac à fumer',
+    };
+    return names[type] || type;
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)